src/um/wincodec.rs

```diff
@@ -5,295 +5,720 @@
+    WICPngChrmRedX = 0x00000003,
+    WICPngChrmRedY = 0x00000004,
+    WICPngChrmGreenX = 0x00000005,
+    WICPngChrmGreenY = 0x00000006,
+    WICPngChrmBlueX = 0x00000007,
+    WICPngChrmBlueY = 0x0000008,
+    WICPngChrmProperties_FORCE_DWORD = CODEC_FORCE_DWORD,
 }}
+
 ENUM!{enum WICPngHistProperties {
-    WICPngHistFrequencies = 0x1,
+    WICPngHistFrequencies = 0x00000001,
+    WICPngHistProperties_FORCE_DWORD = CODEC_FORCE_DWORD,
 }}
+
 ENUM!{enum WICPngIccpProperties {
-    WICPngIccpProfileName = 0x1,
-    WICPngIccpProfileData = 0x2,
+    WICPngIccpProfileName = 0x00000001,
+    WICPngIccpProfileData = 0x00000002,
+    WICPngIccpProperties_FORCE_DWORD = CODEC_FORCE_DWORD,
 }}
+
 ENUM!{enum WICPngSrgbProperties {
-    WICPngSrgbRenderingIntent = 0x1,
+    WICPngSrgbRenderingIntent = 0x00000001,
+    WICPngSrgbProperties_FORCE_DWORD = CODEC_FORCE_DWORD,
 }}
+
 ENUM!{enum WICPngTimeProperties {
-    WICPngTimeYear = 0x1,
-    WICPngTimeMonth = 0x2,
-    WICPngTimeDay = 0x3,
-    WICPngTimeHour = 0x4,
-    WICPngTimeMinute = 0x5,
-    WICPngTimeSecond = 0x6,
+    WICPngTimeYear = 0x00000001,
+    WICPngTimeMonth = 0x00000002,
+    WICPngTimeDay = 0x00000003,
+    WICPngTimeHour = 0x00000004,
+    WICPngTimeMinute = 0x00000005,
+    WICPngTimeSecond = 0x00000006,
+    WICPngTimeProperties_FORCE_DWORD = CODEC_FORCE_DWORD,
 }}
+
 ENUM!{enum WICSectionAccessLevel {
-    WICSectionAccessLevelRead = 0x1,
-    WICSectionAccessLevelReadWrite = 0x3,
+    WICSectionAccessLevelRead = 0x00000001,
+    WICSectionAccessLevelReadWrite = 0x00000003,
+    WICSectionAccessLevel_FORCE_DWORD = CODEC_FORCE_DWORD,
 }}
+
 ENUM!{enum WICPixelFormatNumericRepresentation {
-    WICPixelFormatNumericRepresentationUnspecified = 0,
-    WICPixelFormatNumericRepresentationIndexed = 0x1,
-    WICPixelFormatNumericRepresentationUnsignedInteger = 0x2,
-    WICPixelFormatNumericRepresentationSignedInteger = 0x3,
-    WICPixelFormatNumericRepresentationFixed = 0x4,
-    WICPixelFormatNumericRepresentationFloat = 0x5,
+    WICPixelFormatNumericRepresentationUnspecified = 0x00000000,
+    WICPixelFormatNumericRepresentationIndexed = 0x00000001,
+    WICPixelFormatNumericRepresentationUnsignedInteger = 0x00000002,
+    WICPixelFormatNumericRepresentationSignedInteger = 0x00000003,
+    WICPixelFormatNumericRepresentationFixed = 0x00000004,
+    WICPixelFormatNumericRepresentationFloat = 0x00000005,
+    WICPixelFormatNumericRepresentation_FORCE_DWORD = CODEC_FORCE_DWORD,
 }}
+
 ENUM!{enum WICPlanarOptions {
-    WICPlanarOptionsDefault = 0,
-    WICPlanarOptionsPreserveSubsampling = 0x1,
+   WICPlanarOptionsDefault = 0x00000000,
+   WICPlanarOptionsPreserveSubsampling = 0x00000001,
+   WICPLANAROPTIONS_FORCE_DWORD = CODEC_FORCE_DWORD,
+}}
+
+ENUM!{enum WICJpegIndexingOptions {
+   WICJpegIndexingOptionsGenerateOnDemand = 0x00000000,
+   WICJpegIndexingOptionsGenerateOnLoad = 0x00000001,
+   WICJpegIndexingOptions_FORCE_DWORD = CODEC_FORCE_DWORD,
```

Comment:
Why is this indented by only 3 spaces? I see several cases of this across this file.